xml/System.Text.Encodings.Web/JavaScriptEncoder.xml

```diff
@@ -52,10 +52,10 @@
         <Parameter Name="settings" Type="System.Text.Encodings.Web.TextEncoderSettings" />
       </Parameters>
       <Docs>
-        <param name="settings">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="settings">Settings used to control how the created <see cref="JavaScriptEncoder"/> encodes, primarily which characters to encode.</param>
+        <summary>Creates a new instance of JavaScriptEncoder with provided settings.</summary>
```

Comment:
of JavaScriptEncoder --> of the JavaScriptEncoder class
provided settings --> the specified settings